Azure/azure-cli
# [AMS] Fix #14687: Mixed resource group and account name in command "az ams streaming-endpoint show" #14687

**Description**  

Switched resource group and account name in output string.

**Testing Guide**  

Run the following command with an incorrect streaming endpoint name to see the error string that was changed
`az ams streaming-endpoint show --resource-group [rg]--account-name [accountName]--name [incorrectName]`

**History Notes**  


[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).
